src/modules/startApp.js

```diff
@@ -128,22 +137,68 @@ const createTaskElem = (task) => {
   return listItem;
 };
 
+//-----------------------------------------
+
+// change task status
+const changeTaskStatus = (e) => {
+  let divContainer = null;
+  if (e.target.dataset.index) {
+    divContainer = e.target;
+  } else {
+    divContainer = e.target.parentElement;
+  }
+  const idx = divContainer.dataset.index;
+  const task = tasks[idx - 1];
+  task.completed = !task.completed;
+  divContainer.parentElement.classList.toggle('completed');
+  saveTasksOnLocalStorage();
+};
+
+// create event listener on checkbox and select task
+const addTaskChangeStatusEvent = (listItemElem) => {
+  const divStatElem = listItemElem.querySelector('.status');
+  divStatElem.onclick = changeTaskStatus;
+};
+
 // display task on DOM and ADD EVENT LISTENERS to TASK
 const displayTaskElem = (task) => {
   const listItemElem = createTaskElem(task);
   todoContainerElem.appendChild(listItemElem);
   taskOnFocusListener(listItemElem);
   taskFocusOutListener(listItemElem);
+  addTaskChangeStatusEvent(listItemElem);
+  // testing on drag end
+  listItemElem.ondragstart = (ev) => {
+    ev.dataTransfer.setData('id', ev.target.id);
+  };
 };
 
 const displayAllTasks = () => {
   getTasksFromLocalStorage();
-  tasks.sort((a, b) => a.index - b.index);
-  tasks.forEach((task) => {
+  tasks.forEach((task, i) => {
+    task.index = i + 1;
     displayTaskElem(task);
   });
+  // resetTasksIndexes();
+};
+
+const removeAllTasksFromDom = () => {
+  todoContainerElem.innerHTML = '';
 };
 
+// create event listener to clear all completed tasks
+const clearAllCompletedTasks = () => {
```

Comment:
- [ ] The function(`button`) to clear all complemented tasks is not working, please kindly fix the issue as it is a requirement for this project.

Reply:
@t-yanick Right! 😳 I put the functionality in the little reload icon on the top right corner. I will also add it to the main button 👍